packages/survey-angular-ui/src/components/svg-icon/svg-icon.component.ts

```diff
@@ -45,7 +45,7 @@ export class SvgIconComponent implements OnChanges {
   ngOnChanges(): void {
     const el = this.viewContaierRef.element.nativeElement;
     el.innerHTML = "";
-    el.appendChild(document.createElementNS("http://www.w3.org/2000/svg", "use"));
+    el.appendChild(Survey.settings.environment.createElementNS("http://www.w3.org/2000/svg", "use"));
```

Comment:
Seems to me it is unnecessary changes, we can create element from document and append it to shadow dom.